testsuite/tests/backtrace/backtrace_or_exception.ml

```diff
@@ -0,0 +1,54 @@
+(* TEST
+   flags = "-g"
+   ocamlrunparam += ",b=1"
+   * bytecode
+     reference = "${test_source_directory}/backtrace_or_exception.byte.reference"
```

Comment:
This line is over 80 chars!

Reply:
Not anymore! :)

Reply:
I thank you - now that the an off-by-one error in the Travis CI script is fixed, it'll even get scanned next time 🙂